instana/span.py

```diff
@@ -262,6 +261,10 @@ def __init__(self, span, source, service_name, **kwargs):
         if "rabbitmq" in self.data and self.data["rabbitmq"]["sort"] == "consume":
             self.k = 1  # entry
 
+        # unify the span operation_name for gcps-producer and gcps-consumer
+        if "gcps" in self.data:
```

Comment:
I believe an operation name would be more reliable option.

Reply:
I'll use `span.operation_name` then